example_project/karma.conf.ts

```diff
@@ -162,7 +162,7 @@ const browsers = {
     platform: 'iOS',
     osVersion: '17',
     browserName: 'Safari',
-    useHttps: false,
+    useHttps: true,
```

Comment:
Please also adjust the DOM test that excludes Safari 17 from the HTTPS check.

BTW maybe the fix also helped with desktop Safari. Did you check it?

Reply:
Looks like it didn't